index.js

```diff
@@ -38,22 +38,30 @@ class PageData {
     }
     const res = await wreck.request(method, url, reqOpts);
 
-    if (res.statusCode === 404) {
-      throw Boom.notFound();
+    if (res.statusCode >= 400) {
```

Comment:
can we do this after the read like in wreck - that way we can pass the payload back